### PR TITLE
DDF-2721: Replaced NPM with Yarn

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Switches out NPM in favor of Yarn.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@brendan-hofmann @djblue @ahoffer 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Build](https://github.com/orgs/codice/teams/build)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### How should this be tested? (List steps with links to updated documentation)
All OS: Complete build works
Windows: Quickbuild with no npm cache or yarn cache and all node_modules folders deleted is successful
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
Have you looked into the build failures at all?

File: catalog/ui/catalog-ui-search/package.json [hunk 1]
```diff
@@ -70,8 +70,8 @@
     "webpack-merge": "0.14.0"
   },
   "dependencies": {
-    "@turf/circle": "3.3.3",
```
Comment (pklinef) [inline catalog/ui/catalog-ui-search/package.json]: I am concerned about potential regressions from lowering the turf version.  How was this regression tested?

It looks like public scoped packages should work with Yarn.
https://github.com/yarnpkg/yarn/issues/825

Maybe we can try adding the namespace to the npmrc file for this project.
https://github.com/yarnpkg/yarn/pull/839
`@turf:registry=http://registry.npmjs.org/`

File: catalog/ui/catalog-ui-search/src/main/webapp/js/model/Metacard.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ define([
         'terraformer',
         'terraformer-wkt-parser',
         'js/CQLUtils',
-        '@turf/turf',
```
Comment (pklinef) [inline catalog/ui/catalog-ui-search/src/main/webapp/js/model/Metacard.js]: Please revert these back.

File: catalog/ui/search-ui/standard/README.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Automated tests can execute against a local Selenium server and target locally i
 * Install and start a local Selenium server.  [Webdrvr](https://www.npmjs.org/package/webdrvr) can automate this process.
 
 ```
-npm install -g webdrvr
+yarn install -g webdrvr
```
Comment (djblue) [inline catalog/ui/search-ui/standard/README.md]: I think you need to do `yarn global add webdrvr`

https://yarnpkg.com/en/docs/cli/global

Comment (AzGoalie) [inline catalog/ui/search-ui/standard/README.md]: It was recently deprecated, will update.

**Comment by brendan-hofmann** [comment]
Passed a full build on windows. Building on linux and then will do manual verification.

**Pull request merged**